### ISSUE TITLE
Immediate uninstall if repository casing is mismatched

### ISSUE THREAD

**Issue by neothemachine** [jspm/github#93]
If you do `jspm install github:alice-and-bob/have-fun` where the username is actually called `Alice-and-Bob` then jspm happily installs the package but one step after doing so removes the "package configuration" again.

**Comment by guybedford** [comment]
Is this with jspm 0.16 or jspm 0.17?

**Comment by guybedford** [comment]
Also see https://github.com/jspm/jspm-cli/issues/1274.

**Comment by neothemachine** [comment]
That's 0.16.

**Comment by adamburgess** [comment]
This also happens for .17 (I recently renamed my github and had to do a hard delete and jspm install to fix it)

**Comment by guybedford** [comment]
I guess the simple thing to do here is just to add a `toLowerCase` on the comparison filter in the `clean` method after installing. But casing in general is still pretty badly handled...

**Comment by guybedford** [comment]
Odd I can't seem to recreate this now... I just tried `jspm install github:components/classlist.js` in both 0.16 and 0.17 and it didn't seem to remove it in either.

**Comment by adamburgess** [comment]
My issue was the username -- AdamBurgess to adamburgess.

Also, it was on Windows, ie case insensitive. Not sure if that makes a difference or not. I'll test again shortly

**Comment by guybedford** [comment]
Help replicating here would still be appreciated.

**Comment by guybedford** [comment]
(or did we fix this casing stuff already?)

**Comment by adamburgess** [comment]
@guybedford I've just retried this on .17 beta 28 and .16 and it's fixed for me, though installing something like AdamBurgess/x is saved as AdamBurgess/x, not the "correct" adamburgess/x

maybe this would be fixed with https://github.com/jspm/github/issues/54 ?

**Comment by guybedford** [comment]
@adamburgess yes casing adjustments should be treated as a redirect actually, which should be relatively straightforward to add.
